LLVM_LICENSE

```diff
@@ -1,5 +1,5 @@
-Some derivative works from the LLVM Project may be found in camp.
-The full text of that license is included below.
+Some derivative works from the LLVM Project exist in camp. The following
+licence is referenced in afftected files.
```

Comment:
Spelling: "licence" and "afftected"

Reply:
Good grief....I think I need a nap. Fixes pushed.